testing/cldr-emoji-annotation/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+pkgname=cldr-emoji-annotation
+pkgver=33.1.0.0
+_tag=33.1.0_0
+pkgrel=0
+pkgdesc="Emoji annotation files in CLDR"
+arch="noarch"
+url="https://github.com/fujiwarat/cldr-emoji-annotation"
+license="custom"
```

Comment:
I guess license should be one of:
Unicode-DFS-2015
Unicode-DFS-2016
Unicode-TOU

see https://spdx.org/licenses/

Can you figure out which it is?